model/resource/oci.yaml

```
@@ -0,0 +1,22 @@
+groups:
+  - id: oci
+    prefix: oci
```

```suggestion
    prefix: oci.manifest
```
I think it makes sense to define the manifest as a "top-level" since we may use other OCI concepts later on? Also it makes defining the attributes below shorter.